packages/core/src/csv-schema/csv-schema-generator.ts

```diff
@@ -62,6 +62,6 @@ export class CsvSchemaGenerator implements ArtefactGenerator {
         documentationIdentifier: string,
         callerContext: unknown
     ): Promise<unknown | null> {
-        return null;
+        return null; //Todo: What is this good for?
```

Comment:
This method is for advanced interaction with documentation generators such as Bikeshed. You can keep it empty for now.

If you are curious, see the JSON schema generator. https://github.com/opendata-mvcr/model-driven-data/blob/e88941dff2d58f3ae46e6005c8971e95f32383e6/packages/core/src/json-schema/json-schema-generator.ts#L63-L82